DiamondLightSource/SynchWeb
# LIMS-1353: Cannot book multiple dewars in one shipment

**JIRA ticket**: [LIMS-1353](https://jira.diamond.ac.uk/browse/LIMS-1353)

**Summary**:
https://github.com/DiamondLightSource/SynchWeb/pull/731 introduced a bug whereby multiple dewars in one shipment were combined together, so only one was booked to ship. Counting the containerids seemed to cause the problem.

**Changes**:
- Group the results by dewarId, so a list of dewars is still returned.

**To test**:
- Create a shipment with >1 dewar, book the AWB, check both dewars have a tracking number, and there are the right number of pages in the AWB.
